index.html

```diff
@@ -6,11 +6,20 @@
         <style type="text/css">
             body { margin:0; }
         </style>
-        <script src="assets/phaser.min.js"></script>
+        <script>
+        var supportsWebGL = ( function () { try { return !! window.WebGLRenderingContext && !! document.createElement( 'canvas' ).getContext( 'experimental-webgl' ); } catch( e ) { return false; } } )();
+        if(supportsWebGL){
+            document.write("<script src=\"assets/phaser.min.js\"><\/script>");
```

Comment:
Because I think the import still belongs in the head, not in the body. I was also following the existing format of where we put our imports.